lib/libmbedtls/mbedtls/include/mbedtls/aes.h

```diff
@@ -1,3 +1,4 @@
+/* SPDX-License-Identifier: Apache-2.0 */
```

Comment:
There should be a single SPDX identifier per file.
If you don't want to edit all the files manually you can use my script:
```
$ curl https://raw.githubusercontent.com/jforissier/misc/master/spdxify.py >/tmp/spdxify.py && chmod +x /tmp/spdxify.py
$ /tmp/spdxify.py -k --strip-spdx --add-spdx lib/libmbedtls
```

Reply:
Thanks for the script. Before I run it.
Do we really care about this?
And more importantly, are we permitted to do this?

Reply:
> Do we really care about this?

I think we do, because:

1. the whole purpose of defining a strict format and location for the SPDX line is to avoid the mess (variants) and thus help figure out what we have as simply as possible (a simple grep should do). If we allow multiple occurrences of the SPDX line, some formatted differently, it doesn't work so well anymore.

2. A more convincing argument perhaps is that I think multiple SPDX lines are not allowed by the spec, although I can't find a definitive reference. The text in https://spdx.org/spdx-specification-21-web-version certainly shows a single line and when there are multiple licenses also.

> And more importantly, are we permitted to do this?

Obviously IANAL but since we're effectively just moving the tag up a few lines, it must be OK.

Lastly, since this cleaning does not involve much work at all, it seems reasonable to do.